### ISSUE TITLE
Will be sensors supported?

### ISSUE THREAD

**Issue by michapr** [ripper121/ESPRCSwitch#1]
I read only about switches - how about sensors?
If you are using modifies ESPiLight version - it could be possible, right?

Is the full source code available?  - then I could work on it, maybe.

Thanks,
Michael

**Comment by ripper121** [comment]
Yes you can also read out sensors, but you need to implement it by your own, every sensor sends out different Signals.

THe ESPRCSwicht UI is based on this:
https://github.com/puuu/ESPiLight
and this is based on 
https://www.pilight.org/
